Hunting Queries/SecurityEvent/Least_Common_Parent_Child_Process.txt

```diff
@@ -0,0 +1,19 @@
+// Name: Least Common Parent And Child Process Pairs
+// Id: 3712595d-6f47-416b-963a-605201ed2764
+// Description: Looks across your environment for least common Parent/Child process combinations.  Will possibly find some malicious activity disguised as well known process names.  By ZanCo
+// DataSource: #SecurityEvent
+// Tactics: #InitialAccess, #Execution, #Persistance
+let Whitelist = dynamic (['foo.exe', 'baz.exe']);
+let Sensitivity = 5;
+let StartDate = ago(7d);
+let Duration = 7d;
+SecurityEvent
+| where EventID == 4688 and TimeGenerated > StartDate and TimeGenerated < (StartDate + Duration)
```

Comment:
As the goal here is parent child pairs, potentially drop any events that do not have the parent available:
| where isnotempty(ParentProcessName)

Reply:
Also may want to exclude conhost.exe right off the top - NewProcessName !has "conhost.exe" - or something similar.

Reply:
done